changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## `v0.16.0`
+
+### Features:
+
+- Added ability to pass a genesis file and peer address to `starport network chain join`
+- Added ability to show genesis or peer list with a flag in `starport network chain show`
+- `protoc` is now bundled with Starport CLI
```

Comment:
what's the benefit of bundling protoc?

Reply:
Good question! With `protoc` bundled you don't have to install it manually, and Starport no longer has any dependencies, except Go.

Reply:
This has been the biggest pain point when trying to install Starport.